### DISCUSSION TITLE
Any hope for Oracle?

### DISCUSSION THREAD

**Discussion by twoxfh** [sqlpage/SQLPage#159] [Ideas]
I am curious if there is any support or intentions to support Oracle? I know Oracle Apex competes in the space, always nice to have options. I really like how quickly you can mock an endpoint with the json component and Oracle has pretty strong support in 19c+. Thoughts?

**Comment by lovasoa** [comment]
Hello !
  The underlying database driver library that SQLPage uses to connect to databases is [sqlx-oldapi](https://github.com/lovasoa/sqlx-oldapi). It currently supports four database protocols: MySQL, PostgreSQL, MSSQL, and SQLite, which are all supported in SQLPage. If someone adds support for Oracle to sqlx-oldapi, then I'll use it SQLPage. However, it's currently not in my plans to do it myself :)

**Reply by twoxfh** [reply]
Got it, thanks. I will take a look, maybe others will be interested as well.
